File: hw/sanchez/u3/hw21FarmSystem/AnimalFarmSystemJs/model/Food.js
class Food {
    constructor(id, description) {
        this.id = id;
        this.description = description;
    }

    getId() { return this.id; }
    setId(id) { this.id = id; }
    getDescription() { return this.description; }
    setDescription(description) { this.description = description; }
}

module.exports = Food;

File: hw/sanchez/u3/hw21FarmSystem/AnimalFarmSystemPy/model/food.py
class Food:
    def __init__(self, id, description):
        self._id = id
        self._description = description

    def get_id(self): return self._id
    def set_id(self, id): self._id = id
    def get_description(self): return self._description
    def set_description(self, description): self._description = description